pkg/scheduler/internal/cache/cache.go

```diff
@@ -751,17 +753,18 @@ func (cache *cacheImpl) cleanupAssumedPods(now time.Time) {
 	for key := range cache.assumedPods {
 		ps, ok := cache.podStates[key]
 		if !ok {
-			klog.ErrorS(nil, "Key found in assumed set but not in podStates, potentially a logical error")
+			logger.Error(nil, "Key found in assumed set but not in podStates, potentially a logical error")
 			klog.FlushAndExit(klog.ExitFlushTimeout, 1)
+			// os.Exit(1)
```

Comment:
?

Reply:
The previous PR legacy, sorry, I did not notice this, next time I will modify together.